### PR TITLE
Feature/325 Pull Cash On Hand Forward

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
[App#325](https://app.zenhub.com/workspaces/fecfile-online-619e578e68408b001c831251/issues/gh/fecgov/fecfile-web-app/325)

### CODE REVIEW

File: django-backend/fecfiler/reports/models.py [hunk 1]
```diff
@@ -42,6 +42,11 @@ class Report(SoftDeleteModel, CommitteeOwnedModel):
     treasurer_suffix = models.TextField(null=True, blank=True)
     date_signed = models.DateField(null=True, blank=True)
     calculation_status = models.CharField(max_length=255, null=True, blank=True)
+    calculation_token = models.UUIDField(
```
Comment (mjtravers) [inline django-backend/fecfiler/reports/models.py:45]: Unless it is documented elsewhere and I missed it, it'd be good to put a brief inline comment on what the calculation_token is for or how it is used in the calculation process

File: django-backend/fecfiler/web_services/summary/tasks.py [hunk 2]
```diff
@@ -19,175 +21,213 @@ def __str__(self):
...
+    for report in claimed_reports:
+        summary_service = SummaryService(report)
+        summary = summary_service.calculate_summary()
+        a = summary["a"]
+        b = summary["b"]
+
+        # line 6a
+        report.form_3x.L6a_cash_on_hand_jan_1_ytd = b["line_6a"]
+        # line 6b
+        report.form_3x.L6b_cash_on_hand_beginning_period = a["line_6b"]
```
Comment (mjtravers) [inline django-backend/fecfiler/web_services/summary/tasks.py:61]: Is there a reason I'm missing that the getter ( i.e. a.get("line_6b", 0) ) is not being used on lines 59 and 61?
Do we want a None result in some scenarios for COH?

**Review by mjtravers — COMMENTED** [review]
A couple of inline comments in the review.

File: django-backend/fecfiler/reports/models.py (hunk 1 above)
Comment (Elaine-Krauss-TCG) [reply django-backend/fecfiler/reports/models.py]: I added a comment explaining its purpose

File: django-backend/fecfiler/web_services/summary/tasks.py (hunk 2 above)
Comment (Elaine-Krauss-TCG) [reply django-backend/fecfiler/web_services/summary/tasks.py:61]: It's my understanding that we do sometimes want to have a None result for COH.  If I remember correctly, *in the front end* we ask the user to provide Cash on Hand if all reports within a year have a COH of None.

**Review by toddlees — APPROVED** [review]

**Pull request merged**